ipywidgets/widgets/tests/test_widget_datetime.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Vidar Tonaas Fauske.
```

Comment:
We probably want to follow the other copyright headers? (caught the eye while glancing at the diff)

```suggestion
# Copyright (c) Jupyter Development Team.
```